docs/notification-workflow.md

```diff
@@ -47,70 +47,79 @@ $manager->notify($notification);
 ### Preparing a notification for display
 
   1. In `app.php` register your Notifier (`\OCP\Notification\INotifier`) interface to the manager,
-  using a `\Closure`:
+  using a `\Closure` returning the Notifier and a `\Closure` returning an array of the id and name:
 ```php
 $manager = \OC::$server->getNotificationManager();
 $manager->registerNotifier(function() {
     return new \OCA\Files_Sharing\Notifier(
         \OC::$server->getL10NFactory()
     );
+}, function() {
+		$l = \OC::$server->getL10N('myapp');
+		return [
+			'id' => 'myapp',
+			'name' => $l->t('My apps name'),
+		];
 });
 ```
 
   2. The manager will execute the closure and then call the `prepare()` method on your notifier.
   If the notification is not known by your app, just throw an `\InvalidArgumentException`,
   but if it is actually from your app, you must set the parsed subject, message and action labels:
 ```php
-protected $factory;
+class Notifier implements OCP\Notification\INotifier {
```

Comment:
I added this to make it more clear which interface needs to be implemented (it is written in step 1 but not in step 2 :/)